worlds/rabi_ribi/__init__.py

```diff
@@ -151,8 +159,21 @@ def create_regions(self) -> None:
         region_helper.configure_slot_data()
         region_helper.configure_region_spoiler_log_data()
 
+    def get_filler_item_name(self) -> str:
+        if self.filler_items is None:
+            self.filler_items = [item for sublist in [([key] * value) for key, value in filler_items.items()] for item in sublist]
```

Comment:
Nit: can we add a comment on what this does? My brain is finding this hard to parse.

Reply:
This creates a list of lists and flattens it; tbh I hate the list comprehensions way as well, so I switched to using `itertools.chain()` (it's faster anyway)